src/wizard/__tests__/wizard.test.tsx

```diff
@@ -407,7 +407,7 @@ test('raises a warning when setting activeStepIndex without onNavigate listener'
 test('does not perform navigation when used in controlled mode', () => {
   const [wrapper] = renderDefaultWizard({ activeStepIndex: 1, onNavigate: () => {} });
   const checkStep = () => {
-    expect(wrapper.findContent()!.getElement()).toHaveTextContent(DEFAULT_STEPS[1].content as string);
+    expect(wrapper.findContent()!.getElement()).toHaveTextContent('Step 2, substep oneStep 2, substep two');
```

Comment:
What this change is for? The content of the updated DEFAULT_STEPS seems as still matching the output.

Reply:
The DEFAULT_STEPS content is now JSX, which means that we can't directly compare it anymore with the text content that is rendered in the DOM.

Reply:
got it 👍